docs/index.rst

```diff
@@ -26,3 +26,6 @@ The source code and documentation are hosted on
    contributor_guide/index
 
    community/index
+
+We also have a collection of :ref:`frequently asked questions <faq>` about
```

Comment:
This file won't exist anymore once my PR goes in, which should be today or first thing tomorrow. It is then called contents.rst because index is generated from the html template (landing page).

Reply:
I shall fix the conflict once your change is in :-)